torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -2111,10 +2111,31 @@ Tensor symeig_backward(const std::vector<torch::autograd::Variable> &grads, cons
 
   auto vh = v.conj().transpose(-2, -1);
 
+  // Based on a little experiment, min_threshold should be
+  //   "max_eigenvalue * matrix_size * machine_eps"
+  // See: https://github.com/pytorch/pytorch/pull/50942
+  // However, as getting max_eigenvalue would need synchronization, I just take
+  //   a guess that the max eigenvalues would typically be less than 10
+  // Eps is still taking double and single eps value, because I don't know
+  //   if it would produce a lot of backward incompatibility due to large
+  //   eps for half-precision floating point
+  double eps = (lambda.dtype() == at::kDouble) ? 2.220446049250313e-16 : 1.1920928955078125e-07;
```

Comment:
I think `std::numeric_limits<T>::epsilon()` should be used here.
There is also `_get_epsilon` from `ATen/native/LinearAlgebraUtils.h`.

Reply:
Thanks! I've changed it `std::numeric_limits` in the new commit.